code/game/area/areas/misc.dm

```diff
@@ -23,9 +23,6 @@
 /area/space/nearstation
 	icon_state = "space_near"
 	area_flags = UNIQUE_AREA | AREA_USES_STARLIGHT
-	static_lighting = TRUE
-	base_lighting_alpha = 0
-	base_lighting_color = COLOR_WHITE
```

Comment:
This is how nearspace looks like after this change

<details>
  <summary>Spoiler warning</summary>

  ![image](https://github.com/tgstation/tgstation/assets/7501474/f7f407e9-63b8-4c65-897d-748e033a2b5d)
  
</details>